galite-core/src/main/kotlin/org/kopi/galite/visual/KopiAction.kt

```diff
@@ -0,0 +1,54 @@
+/*
+ * Copyright (c) 2013-2020 kopiLeft Services SARL, Tunis TN
+ * Copyright (c) 1990-2020 kopiRight Managed Solutions GmbH, Wien AT
+ *
+ * This library is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License version 2.1 as published by the Free Software Foundation.
+ *
+ * This library is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public
+ * License along with this library; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301  USA
+ */
+
+package org.kopi.galite.visual
+
+import kotlinx.coroutines.Runnable
+
+abstract class KopiAction() : Runnable {
+  lateinit var name: String
+
+  constructor(name: String) : this() {
+    this.name = name
+  }
+
+  @Throws(VException::class)
+  abstract fun execute()
+
+  override fun run() {
+    try {
+      execute()
+    } catch (e: VException) {
+      throw e.message?.let { VRuntimeException(it, e) }!!
```

Comment:
e.message.let { VRuntimeException(it, e) }
And change in VRuntimeException to accept null in the called constructor